src/frontends/tensorflow_common/src/op/tensor_list_operations.cpp

```diff
@@ -122,6 +122,38 @@ OutputVector translate_tensor_list_set_item_op(const NodeContext& node) {
     return {scatter_update};
 }
 
+OutputVector translate_tensor_list_push_back_op(const NodeContext& node) {
+    default_op_checks(node, 2, {"TensorListPushBack"});
+    auto input_handle = node.get_input(0);
+    auto tensor = node.get_input(1);
+
+    // flatten item to be inserted since
+    // the tensor list saves elements in the flatten form
+    // because we want to cover a case of dynamic rank tensor list
+    // the real shape of the tensor elements will be restored by TensorListStack operations
+    auto new_tensor_shape = make_shared<Constant>(element::i32, Shape{1}, -1);
+    tensor = make_shared<Reshape>(tensor, new_tensor_shape, false);
+    auto tensor_shape = make_shared<ShapeOf>(tensor, element::i32);
+
+    // reshape the tensor list to the shape [num_elements, -1]
+    // that is because in the first iteration we have empty constant of a shape [0,0]
+    Output<Node> num_elements = make_shared<ShapeOf>(input_handle, element::i32);
+    auto zero_const = make_shared<Constant>(element::i32, Shape{1}, 0);
```

Comment:
I remember we have such constants (zero_const, one_const, two_const, minus_one_const?) in another translators. I saw it in convolution, for example. I didn't check other places.

If it is a common constant I suggest to move it to common constant.